Frameworks/UIKit/UIFont.mm

```diff
@@ -856,102 +321,33 @@ - (instancetype)copyWithZone:(NSZone*)zone {
     return [self retain];
 }
 
-/**
- @Status Interoperable
-*/
-- (void)dealloc {
-    _fileName = nil;
-    _name = nil;
-    _descriptor = nil;
-    [super dealloc];
-}
-
 - (void)_setName:(NSString*)name {
-    _fileName = name;
+    UNIMPLEMENTED();
 }
 
 - (uint32_t)_fontHandle {
-    return (uint32_t)_font;
+    UNIMPLEMENTED();
+    return StubReturn();
 }
 
 - (uint32_t)_sizingFontHandle {
-    return (uint32_t)_sizingFont;
+    UNIMPLEMENTED();
+    return StubReturn();
 }
 
 // Internal methods
 
 // Private message sent from CTFontManager for the implementation of CTFontManagerRegisterGraphicsFont
 - (bool)_CTFontManagerRegisterGraphicsFont:(CGFontRef)font withError:(CFErrorRef*)error {
```

Comment:
>_CTFontManagerRegisterGraphicsFont [](start = 8, length = 34)

Can we remove these private methods and update their callers instead so we know what public facing APIs have regressed with this?